.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,59 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
This file seems to have been committed accidentally

Reply:
yes sorry about that